src/web/browser/coreVitals/coreVitals.ts

```diff
@@ -62,23 +69,19 @@ export const coreVitals = (): void => {
 		// If CLS has been calculated
 		if (jsonData.cls !== null) {
 			// Set page view and browser ID
-			if (window.guardian && window.guardian.ophan) {
-				jsonData.page_view_id = window.guardian.ophan.pageViewId;
-				jsonData.browser_id = window.guardian.config.ophan.browserId;
 
-				fetch(endpoint, {
-					method: 'POST', // *GET, POST, PUT, DELETE, etc.
-					mode: 'cors', // no-cors, *cors, same-origin
-					cache: 'no-cache', // *default, no-cache, reload, force-cache, only-if-cached
-					credentials: 'same-origin', // include, *same-origin, omit
-					headers: {
-						'Content-Type': 'application/json',
-					},
-					redirect: 'follow',
-					referrerPolicy: 'no-referrer', // no-referrer, *no-referrer-w
-					body: JSON.stringify(jsonData),
-				}).catch(() => {});
-			}
+			fetch(endpoint, {
```

Comment:
So this now means that we'll send the data even if ophan hasn't loaded? (but with nulls for pageview and browser ID

Reply:
No, it still has the CSL check on line 70